vyxal/LazyList.py

```diff
@@ -267,3 +267,16 @@ def reversed(self):
         self.generated += list(itertools.tee(self.raw_object)[-1])
         for item in self.generated[::-1]:
             yield item
+
+    def remove(self, value):
```

Comment:
Weren't we were planning on making `LazyList`s immutable? This mutates the generated list.